src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -249,7 +249,7 @@ function compute_flux_differencing_SBP_matrices(dg::DGMulti, sparse_operators)
   rd = dg.basis
   Qrst_hybridized, VhP, Ph = StartUpDG.hybridized_SBP_operators(rd)
   Qrst_skew = map(A -> 0.5 * (A - A'), Qrst_hybridized)
-  if sparse_operators isa Val{true}
+  if sparse_operators == true
```

Comment:
Will this work? It seems like `Static.True` isn't the same as `true`?
```julia
julia> using Static
julia> True == true
false
```

Reply:
We do not compare the types but instances of the type:
```julia
julia> using Static

julia> True() == true
true
```

Reply:
Thanks!